aquarius/ddo_checker/shacl_schemas/v4/remote_4.2.0.ttl

```diff
@@ -0,0 +1,428 @@
+        sh:maxCount 64;
+    ] ;.
+
+
+schema:ConsumerParametersShape
+    sh:targetClass schema:ConsumerParameter ;
+    sh:property [
+        sh:path schema:name;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:type;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:label;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:required ;
+        sh:datatype xsd:boolean ;
+        sh:minCount 1;
+        sh:maxCount 1;
+    ] ;
+    sh:property [
+        sh:path schema:description ;
+        sh:datatype xsd:string ;
+        sh:pattern "^((.|\n)*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 65536;
+    ] ;
+    sh:property [
+        sh:path schema:default;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:options;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:maxLength 256;
+    ] ;.
+
+
+schema:ContainerShape
+    sh:targetClass schema:Container ;
+    sh:property [
+        sh:path schema:entrypoint ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:image ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:tag ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:checksum ;
+        sh:datatype xsd:string ;
+        sh:pattern "^sha256\\:(.*)$" ;
+        sh:maxLength 71;
+        sh:minLength 71;
+        sh:minCount 1;
+        sh:maxCount 1;
+    ] ;.
+
+
+schema:ServiceShape
+    sh:targetClass schema:Service ;
+    sh:property [
+        sh:path schema:id ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:type ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:name ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:maxCount 1;
+        sh:maxLength 256;
```

Comment:
Perhaps this should also have a `minLength`. In the market, the minimum length of a title is set to 4 characters